sycl/test-e2e/Complex/sycl_complex_literals_test.cpp

```diff
@@ -0,0 +1,24 @@
+// DEFINE: %{mathflags} = %if cl_options %{/clang:-fno-fast-math%} %else %{-fno-fast-math%}
+
+// RUN: %{build} -fsycl-device-code-split=per_kernel %{mathflags} -o %t.out
+// RUN: %{run} %t.out
+
+#include "sycl_complex_helper.hpp"
+
+int main() {
+  using namespace sycl::ext::oneapi::experimental::complex_literals;
+
+  bool test_passes = true;
+
+  test_passes &=
```

Comment:
Test for `il` suffix seem to be missing

Reply:
To align with changes, the long double is emitted.